libs/ngx-charts-on-fhir/project.json

```diff
@@ -8,8 +8,7 @@
     "build": {
       "executor": "@nx/angular:package",
       "options": {
-        "project": "libs/ngx-charts-on-fhir/ng-package.json",
-        "updateBuildableProjectDepsInPackageJson": true
```

Comment:
`updateBuildableProjectDepsInPackageJson`   is deprecated so removed it, alternatively added `@nx/dependency-checks` ESLint rule to manage dependency-checks
https://dev.to/this-is-learning/manage-nx-library-dependencies-with-the-nxdependency-checks-eslint-rule-2lem